src/AbstractRoboFile.php

```diff
@@ -1033,11 +1033,12 @@ protected function clearOpCacheTask($worker, AbstractAuth $auth, $remote)
     protected function cleanDirsTask($worker, AbstractAuth $auth, $remote)
     {
         $currentProjectRoot = $remote['currentdir'] . '/..';
+        $cleandirLimit = isset($remote['cleandir_limit']) ? max(1, $remote['cleandir_limit']) : '';
         return $this->taskSsh($worker, $auth)
                 ->remoteDirectory($currentProjectRoot, true)
                 ->timeout(30)
-                ->exec('vendor/bin/robo digipolis:clean-dir ' . $remote['releasesdir'] . (isset($remote['cleandir_limit']) ? ':' . $remote['cleandir_limit'] : ''))
-                ->exec('vendor/bin/robo digipolis:clean-dir ' . $remote['backupsdir'] . (isset($remote['cleandir_limit']) ? ':' . $remote['cleandir_limit'] : ''));
+                ->exec('vendor/bin/robo digipolis:clean-dir ' . $remote['releasesdir'] . (isset($remote['cleandir_limit']) ? ':' . $cleandirLimit : ''))
```

Comment:
I suggest you use !empt($cleandirLimit) instead of isset(...)

Reply:
`!empty()`

Reply:
Why have the check _again_ in the exec() call, since it was already done above when initializing $cleandirLimit?

Reply:
0 is in principe een geldige waarde, zo sluiten we dat ook uit

Reply:
@dubgeiser because if it isn't set, we use the defaults set higher up the chain.